src/hud.cpp

```diff
@@ -489,23 +491,90 @@ void drawItemStack(video::IVideoDriver *driver,
 		const ItemStack &item,
 		const core::rect<s32> &rect,
 		const core::rect<s32> *clip,
-		IGameDef *gamedef)
+		IGameDef *gamedef,
+		bool selected,
```

Comment:
remove this 3 bool and use a byteflag or a enum please to have only 1 parameter instead of three